cloudslang-cli/pom.xml

```diff
@@ -332,8 +338,125 @@
+                                        </descriptor>
+                                    </descriptors>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+                <finalName>cslang</finalName>
+            </build>
+        </profile>
+        <profile>
+            <id>content</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-scm-plugin</artifactId>
+                        <version>1.9.4</version>
+                        <executions>
+                            <execution>
+                                <id>download content sl</id>
+                                <phase>process-sources</phase>
+                                <goals>
+                                    <goal>checkout</goal>
+                                </goals>
+                                <configuration>
+                                    <connectionUrl>scm:git:git@github.com:CloudSlang/cs-content.git</connectionUrl>
+                                    <checkoutDirectory>${project.build.directory}/cs-content</checkoutDirectory>
+                                    <scmVersion>${cs-content.revision}</scmVersion>
+                                    <scmVersionType>${cs-content.type}</scmVersionType>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-resources-plugin</artifactId>
+                        <version>3.0.2</version>
+                        <executions>
+                            <execution>
+                                <id>copy-content</id>
+                                <phase>prepare-package</phase>
+                                <goals>
+                                    <goal>copy-resources</goal>
+                                </goals>
+                                <configuration>
+                                    <outputDirectory>${project.build.directory}/cslang-cli</outputDirectory>
+                                    <resources>
+                                        <resource>
+                                            <directory>${project.build.directory}/cs-content</directory>
+                                            <filtering>false</filtering>
+                                            <includes>
+                                                <include>**</include>
+                                            </includes>
+                                            <excludes>
+                                                <exclude>**/ci-env/**</exclude>
```

Comment:
wouldn't it be more robust to specify explicitly what to include?

Reply:
Good point. Will add only what to include.